packages/app/src/utils/serializers.ts

```diff
@@ -36,6 +36,7 @@ export const serializeFundingCycle = (
   configured: fc.configured.toString(),
   cycleLimit: fc.cycleLimit.toString(),
   ballot: fc.ballot,
+  basedOn: fc.basedOn.toString(),
```

Comment:
same thing.  Replace `previous` with `basedOn` in this object instead of adding the property at the end.